### PR TITLE
docs: Improve current quickstart guides

### PR DESCRIPTION
**Pull request by perkinsjr**
## What does this PR do?

Improves quickstart guides for Next.js, Bun, Express and adds Hono

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [X] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [Contributing Guide](./CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand areas
- [ ] Ran `pnpm build`
- [ ] Ran `pnpm fmt`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

### CODE REVIEW

File: apps/docs/mint.json [hunk 1]
```diff
@@ -136,11 +141,7 @@
         },
         {
           "group": "Semantic cache",
-          "pages": [
-            "semantic-cache/introduction",
-            "semantic-cache/similarity-threshold",
-            "semantic-cache/why-semantic-caching"
-          ]
+          "pages": ["semantic-cache/introduction", "semantic-cache/why-semantic-caching"]
```
Comment (chronark) [inline apps/docs/mint.json:144]: why did you remove the `similarity-threshold`?

File: apps/docs/quickstart/apis/hono.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  -H "Authorization: Bearer <KEY>"
+```
+
+It should return `Hello Hono!"`
+
+</Step>
+
+</Steps>
+
+## What is next?
```
Comment (chronark) [inline apps/docs/quickstart/apis/hono.mdx:134]: fyi, mintlify has a concept of "snippets", where you could put this common footer and then reference it

**Comment by perkinsjr** [comment]
its not a page, its inside another page so the page didn’t exist.

I am aware of the snippets I’m not using them yet till I’m satisfied that is the concept I want to keep

**Comment by perkinsjr** [comment]
Specifically it lives here [https://www.unkey.com/docs/semantic-cache/settings](https://www.unkey.com/docs/semantic-cache/settings)

**Review by chronark — APPROVED** [review]
